tests/smoke-tests/src/test/java/org/apache/activemq/artemis/tests/smoke/jmx/JmxConnectionTest.java

```diff
@@ -27,8 +27,8 @@
 import java.rmi.server.RemoteRef;
 
 import io.netty.util.internal.PlatformDependent;
-import io.netty.util.internal.shaded.org.jctools.util.UnsafeAccess;
 import org.apache.activemq.artemis.tests.smoke.common.SmokeTestBase;
+import org.jctools.util.UnsafeAccess;
```

Comment:
@franz1981 what occurs post java 8 here? Same with any internal Unsafe access in the MPSC Queue used

Reply:
@michaelandrepearce I see that JCtools qs are fine to use Unsafe until Java 11 (included) for both perf and compatibility (I know is weird) reasons :+1:

Reply:
you say until 11...does that mean after its knackered?

Reply:
We are working on a VarHandle version and we already allows different AtomicXXX versions to be used (with a small perf drop): probably java 12 is fine too, but I haven't tested it personally...
The problem will be when some Unsafe specific methods will be scattered around VarHandle/Other classes and will be officially deleted from Unsafe...